backend/api/Controllers/ParcelController.cs

```diff
@@ -129,12 +129,12 @@ public IActionResult AddParcel([FromBody] Model.ParcelModel model)
         /// <param name="id"></param>
         /// <param name="model"></param>
         /// <returns></returns>
-        [HttpPut("{id}")]
+        [HttpPut()]
```

Comment:
Question: is there an specific reason why the `id` param was removed here? I know it is also supplied as part of the POST body (and might seem counter-intuitive) but that's how Rest UPDATE operations are modelled:

`PUT /api/whatever-entities/5` vs `PUT /api/whatever-entities/`

Reply:
Took it out because it was being ignored by the service, ie. PUT/5 with a body of { id: 10 } would target the entity with id of 10.